# Generated by Django 4.1.1 on 2022-09-21 14:34

import django.contrib.auth.validators
import django.core.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('accounts', '0005_alter_myuser_first_name_alter_myuser_last_name_and_more'),
    ]

    operations = [
        migrations.AlterField(
            model_name='myuser',
            name='username',
            field=models.CharField(error_messages={'unique': 'A user with that username already exists'}, max_length=100, unique=True, validators=[django.contrib.auth.validators.ASCIIUsernameValidator(), django.core.validators.MinLengthValidator(2), django.core.validators.ProhibitNullCharactersValidator], verbose_name='Username'),
        ),
        migrations.DeleteModel(
            name='address',
        ),
    ]
